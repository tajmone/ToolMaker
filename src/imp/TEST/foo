Nu skriver jag till foo
ett par rader.
