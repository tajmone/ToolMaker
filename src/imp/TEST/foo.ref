%%IF (foo(fum))
hej%%foohopp!
%%ENDIF
%%IF (%%foo(fum))
Error!
%%ENDIF
