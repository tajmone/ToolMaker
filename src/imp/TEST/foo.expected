########## foo ##########
%%SET foo("Hej", Hopp)
%%IF (foo(Hej))
Error!
%%ELSE
OK!
%%ENDIF
%%IF (foo("Hopp"))
Error!
%%ELSE
OK!
%%ENDIF
########## ../imp foo.tst >foo.tst.tmp; cat foo.tst.tmp; rm foo.tst.tmp ##########
Error!
Error!
